### ISSUE TITLE
Validators: validate_enum Always Fails

### ISSUE THREAD

**Issue by aubricus** [jazzband/django-eav2#42]
`validate_enum` (see [validators.py](https://github.com/makimo/django-eav2/blob/0.12.0/eav/validators.py) always fails as `value` is always a string. 

## Expected Behavior
`valildate_enum` should not throw a ValidationError when handed a valid pk for EnumValue

## Actual Behavior
`validate_enum` throws a ValidationError when handed a valid pk for EnumValue

## Possible Fix

> This validator is actually disabled in django-eav (https://github.com/mvpdev/django-eav/blob/master/eav/validators.py#L99-L111)

I've addressed this in my own build like so:

```python
def validate_enum(value):
    from .models import EnumValue

    try:
        instance = EnumValue.objects.get(pk=value)
    except EnumValue.DoesNotExist:
        instance = None

    if not instance:
        raise ValidationError("Must be an EnumValue model object instance")
    if instance and not instance.pk:
        raise ValidationError("EnumValue has not been saved yet")
```

## Steps to Reproduce

1. Register a model with eav
2. Create an multiple-choice Attribute (and associated EnumGroup/Value)
3. Create model instance, choose EnumValue for multi-choice Attribute.

## Your Environment

* Version used:
* Django version 2.1.4
* django-eav2==0.11.1 (latest pypi version)
* Linux 2f525468f45c 4.9.125-linuxkit #1 SMP Fri Sep 7 08:20:28 UTC 2018 x86_64 GNU/Linux

**Comment by IwoHerka** [comment]
Thank you for the report. We will take a look at it when possible. :)

**Comment by Dresdn** [comment]
Was resolved in issue #46
